etna/transforms/math/binary_operator.py

```diff
@@ -0,0 +1,166 @@
+from enum import Enum
+from typing import List
+from typing import Optional
+
+import pandas as pd
+
+from etna.datasets import TSDataset
+from etna.transforms.base import ReversibleTransform
+
+
+class BinaryOperator(str, Enum):
+    """Enum for mathematical operators from pandas."""
+
+    add = "+"
+    sub = "-"
+    mul = "*"
+    div = "/"
+    floordiv = "/="
+    mod = "%"
+    pow = "**"
+
+    eq = "=="
+    ne = "!="
+    le = "<="
+    lt = "<"
+    ge = ">="
+    gt = ">"
+
+    @classmethod
+    def _missing_(cls, value):
+        raise ValueError(f"Incorrect operand, literal {value} is unsupported")
+
+    def perform(self, df: pd.DataFrame, left_operand: str, right_operand: str, out_column: str) -> pd.DataFrame:
+        """Perform binary operation on passed dataframe."""
+        pandas_operator = getattr(pd.DataFrame, self.name)
+        df_left = df.loc[:, pd.IndexSlice[:, left_operand]].rename(columns={left_operand: out_column}, level="feature")
+        df_right = df.loc[:, pd.IndexSlice[:, right_operand]].rename(
+            columns={right_operand: out_column}, level="feature"
+        )
+        return pandas_operator(df_left, df_right)
+
+
+class BinaryOperationTransform(ReversibleTransform):
```

Comment:
We should add some info about zeros and `inverse_transform`. You can use `DifferencingTransform` as example how we can do it.